apps/web/src/pages/templates/workflow/WorkflowEditor.tsx

```diff
@@ -17,6 +17,18 @@ import { UpdateButton } from '../components/UpdateButton';
 import { NameInput } from './NameInput';
 import { Settings } from '../../../design-system/icons';
 import { Button } from '../../../design-system';
+import ChannelNode from './workflow/node-types/ChannelNode';
+import TriggerNode from './workflow/node-types/TriggerNode';
+import AddNode from './workflow/node-types/AddNode';
+import { AddNodeEdge } from './workflow/edge-types/AddNodeEdge';
+
+const nodeTypes = {
+  channelNode: ChannelNode,
+  triggerNode: TriggerNode,
+  addNode: AddNode,
+};
+
+const edgeTypes = { special: AddNodeEdge };
```

Comment:
These were hardcoded before in the `FlowEditor` and they are using a lot of different functionalities, like `react-hook-form` etc., but it's only specific to the Template Editor. Now we can define whatever steps we want and we can reuse `FlowEditor`.